tests/test_startup.py

```diff
@@ -6,17 +6,6 @@
 from pytest import fixture, mark, raises
 from sqlalchemy import text
 
-pytestmark = mark.usefixtures("patch_engine_from_config", "patch_new_engine")
-
-
-@fixture(params=[True, False])
-def case_sensitive_environ(environ, request):
```

Comment:
It was causing issue with the `PYTEST_CURRENT_TEST` env var, so moved to its own test